Kursach.js

```diff
@@ -0,0 +1,124 @@
+const Gtk = imports.gi.Gtk;
+const Lang = imports.lang;
+
+Gtk.init(null);
+
+const header = new Gtk.HeaderBar();
+header.set_title('Test');
+header.set_subtitle('Kursach');
+header.set_show_close_button(true);
+
+const win1 = new Gtk.Window();
+win1.window_position = Gtk.WindowPosition.CENTER;
+win1.connect('destroy', Gtk.main_quit);
+win1.set_default_size(750, 270);
+win1.set_titlebar(header);
+win1.border_width = 10;
+win1.show_all();
+
+const ListBoxRowWithData = Lang.Class({
+    Name: "ListBoxRowWithData",
+    Extends: Gtk.ListBoxRow,
+
+    _init: function(data) {
+        this.parent();
+        this.data = data;
+        this.add(new Gtk.Label({label: data}));
+    }
+});
+
+
+
+const ListBoxWindow = Lang.Class({
+    Name: "ListBoxWindow",
+    Extends: Gtk.Window,
+
+    _init: function() {
+        this.parent({title: "Options"});
```

Comment:
```suggestion
        this.parent({ title: 'Options' });
```
Add spaces here and below